kde-apps/calendarjanitor/calendarjanitor-9999.ebuild

```
@@ -31,7 +31,7 @@ RDEPEND="${DEPEND}
 	>=kde-apps/akonadi-calendar-tools-common-${PV}
 "
 
-PATCHES=( "${FILESDIR}/${PN}-24.05.2-loggingcategory.patch" )
+PATCHES=( "${FILESDIR}/${PN}-9999-loggingcategory.patch" )
```

```suggestion
PATCHES=( "${FILESDIR}/${PN}-25.07.70-loggingcategory.patch" )
```

Use the currently set upstream version in CMakeLists.txt. I don't want to have to rename it to make sense when the next Gear version is released.

---

I'm unsure where to find that in this case. I can only find the PIM_VERSION which is used for the project and is a non Gear version.

https://invent.kde.org/pim/akonadi-calendar-tools/

---

Yeah, it's unfortunate in this case where PIM uses its own internal one. But we know the schedule.